openstack-cloud/src/main/resources/index.jelly

```diff
@@ -1,3 +1,3 @@
 <div>
-    Allows Jenkins to build using Cloud Servers via JClouds.
-</div>
\ No newline at end of file
+    Allows Jenkins to build using Cloud Servers in OpenStack via jclouds.
```

Comment:
I'd be fine removing the jclouds reference but left it in due to other occurrences.